npc/dev/test.txt

```diff
@@ -729,6 +729,29 @@ function	script	HerculesSelfTestHelper	{
 	callsub(OnCheckStr, "sprintf (positional)", sprintf("'%2$+05d'", 5, 6), "'+0006'");
 	callsub(OnCheckStr, "sprintf (positional)", sprintf("'%2$s' '%1$c'", "First", "Second"), "'Second' 'F'");
 
+	// unit data test
+	freeloop(1);
```

Comment:
It shouldn't. The `check_gotocount` setting defaults to 2048... If necessary, we should increase it by the way, rather than teaching people to use `freeloop()` everytime they have a small loop.